my.new.project/src/test/java/manualTests.txt

```diff
@@ -0,0 +1,43 @@
+Manual tests
+
+1.T-Can use search bar 
```

Comment:
Use this format:
Title: Can user search bar 
Summary: As a user I can user the search bar ...
Preconditions: 
Steps: 
1. Click the search bar and enter 'svt-810e' 
2.
Expected Results:
A search engine should return results similar to search, with the desired product as the first link 

Requirements not needed here.